### PR TITLE
feat: Move MockERC20 into erc20-helpers (SC-128)

### PR DESCRIPTION
**Pull request by lucas-manuel**
Merge after https://github.com/marsfoundation/erc20-helpers/pull/1

### CODE REVIEW

File: src/MockERC20.sol [hunk 1]
```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.7;
```
Comment (hexonaut) [inline src/MockERC20.sol]: Change to `pragma solidity >=0.8.0;`

File: test/MockERC20.t.sol [hunk 2]
```diff
@@ -0,0 +1,227 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.7;
```
Comment (hexonaut) [inline test/MockERC20.t.sol]: This version can be relaxed too.

**Review by hexonaut — COMMENTED** [review]
Minor points about pragma. Looks good otherwise.

**Review by hexonaut — APPROVED** [review]

**Comment by lucas-manuel** [comment]
@barrutko to review retroactively

**Comment by barrutko** [comment]
Retroactively looks good 🫡

**Pull request merged**